.github/settings.yml

```
@@ -0,0 +1,247 @@
+      enforce_admins: false
+      required_linear_history: false
+      allow_force_pushes: false
+      allow_deletions: false
+
+      restrictions:
+        users: []
+        teams: []
+        apps: []
+
+      # CRITICAL: Allow bots to push without PR
+      required_pull_request_reviews: null
```

YAMLlint (1.38.0)

[error] duplication of key "required_pull_request_reviews" in mapping

(key-duplicates)